src/targets/gpu/fuse_ops.cpp

```diff
@@ -190,11 +189,12 @@ MIGRAPHX_PRED_MATCHER(fusable_conv, instruction_ref ins)
         return false;
     auto wei = ins->inputs().at(1)->get_shape();
     assert(wei.lens().size() == 4);
-    auto miopen_conv_op = any_cast<miopen_convolution<op::convolution>>(ins->get_operator());
-    auto conv_op        = miopen_conv_op.op;
+    auto miopen_conv_op = ins->get_operator().to_value();
+    auto algo           = miopen_conv_op.at("algo").to<int>();
```

Comment:
Do `miopen_conv_op.at("algo").to<miopenConvFwdAlgorithm_t>()`.

Reply:
Done.